### PR TITLE
Lit6535 ders yaq lenten

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: new/LIT6535DersYaqLenten.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">Homily for the sixth Sunday in Lent</title>
```
Comment (eu-genia) [inline new/LIT6535DersYaqLenten.xml:8]: line 8 lang="en" type="main"

File: new/LIT6535DersYaqLenten.xml [hunk 2]
```diff
@@ -7,6 +7,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                 <title xml:lang="gez" corresp="#t1">ድርሳን፡ ዘአባ፡ ያዕቆብ፡ ዘይተነበብ፡ በሳድስ፡ ሰንበተ፡ ጾም።</title>
+                <title xml:lang="gez" corresp="#t1">Dǝrsān za-ʾabbā Yāʿqob za-yǝtnabbab ba-sādǝs sanbata ṣom</title>
```
Comment (eu-genia) [inline new/LIT6535DersYaqLenten.xml:10]: line 10 type="normalized"

File: new/LIT6535DersYaqLenten.xml [hunk 3]
```diff
@@ -52,6 +52,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <text>
         <body>
             <div type="bibliography">
+              <listBibl type="editions" corresp="#t1syriac">
```
Comment (eu-genia) [inline new/LIT6535DersYaqLenten.xml:56]: I do not see the t1syriac title element above. where should the reference point to?
if the Syriac here is irrelevant, then just leave `#t1`, or omit the corresp completely
if this is a bibliography for the Syriac version of the text, you might need to add a t2syriac title for the Syriac version and then point to `#t2Syriac` here

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6535DersYaqLenten.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply new/LIT6535DersYaqLenten.xml:8]: Done

File: new/LIT6535DersYaqLenten.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply new/LIT6535DersYaqLenten.xml:56]: Thank you, I hope it's better now

File: new/LIT6535DersYaqLenten.xml [hunk 4]
```diff
@@ -5,9 +5,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">Homily for the sixth Sunday in Lent</title>
+                <title xml:lang="gez" type="main" xml:id="t1">Homily for the sixth Sunday in Lent</title>
```
Comment (eu-genia) [inline new/LIT6535DersYaqLenten.xml:8]: lang="en"

File: new/LIT6535DersYaqLenten.xml (hunk 2 above)
Comment (eu-genia) [reply new/LIT6535DersYaqLenten.xml:10]: still missing?

**Review by eu-genia — CHANGES_REQUESTED** [review]
thank you!
something slipped or am i missing something?

File: new/LIT6535DersYaqLenten.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply new/LIT6535DersYaqLenten.xml:10]: Done

**Review by eu-genia — APPROVED** [review]

**Pull request merged**